tutorial 2 updates for azure platform and some other typo and cosmetic fixes

balazagi opened the pull request:
- updates for azure platform comparability
- some cosmetic and typo fixes

nhsavage commented:
mostly looking very good - runs for me ok. most of my comments are not about your changes, but the notebook in general 

- Section 1.4 states ". The result of indexing a cube is always a copy of the cube ". Is this true? I thought it was a view?
- Section 1.6 "First define the lat lon on Shanghai region:" -> "First define the lat lon of Shanghai region:"
- Also 1.6 - please print the lat and lon limits before extraction so we have a before and after
- Section 1.7 contrain -> constrain
- Section 2.2 While performing arithmetic calculation, keep into consideration the units -> When performing arithmetic calculation, consider the units
- 3.1 reverse the timeseries plots putting the raw matplolib one first 
- 3.2 We can collapse 'time' dimension os  -> We can collapse 'time' dimension of
- 4.1 # we may wish to change this path! - please open a follow on issue to discuss this (not too bad but could perhaps use $HOME via os.getenv)

balazagi commented:
> mostly looking very good - runs for me ok. most of my comments are not about your changes, but the notebook in general
> 
> * Section 1.4 states ". The result of indexing a cube is always a copy of the cube ". Is this true? I thought it was a view?
> * Section 1.6 "First define the lat lon on Shanghai region:" -> "First define the lat lon of Shanghai region:"
> * Also 1.6 - please print the lat and lon limits before extraction so we have a before and after
> * Section 1.7 contrain -> constrain
> * Section 2.2 While performing arithmetic calculation, keep into consideration the units -> When performing arithmetic calculation, consider the units
> * 3.1 reverse the timeseries plots putting the raw matplolib one first
> * 3.2 We can collapse 'time' dimension os  -> We can collapse 'time' dimension of
> * 4.1 # we may wish to change this path! - please open a follow on issue to discuss this (not too bad but could perhaps use $HOME via os.getenv)

I have addressed the points above where valid as discussed. opened issue #52 for the last point.
